presto-orc/src/main/java/com/facebook/presto/orc/stream/LongInputStreamV1.java

```diff
@@ -34,6 +34,26 @@
     private boolean repeat;
     private long lastReadInputCheckpoint;
 
+    // ========== ARIA FIELDS BEGIN ==========
+    // Position of the first value of the run in literals from the checkpoint.
+    private int currentRunOffset;
+    // Positions to visit in scan(), offset from last checkpoint.
+    private int[] offsets;
+    private int numOffsets;
+    private int offsetIdx;
+    // Copies of arguments to scan().
```

Comment:
This comment is incorrect. Drop it.

Reply:
Removed. Since I copied this from LongInputStreamV2 I am assuming I should remove the comment there as well.